AuthenticatorShared/Core/Platform/Utilities/Constants.swift

```diff
@@ -10,6 +10,9 @@ typealias DeviceType = Int
 enum Constants {
     // MARK: Static Properties
 
+    /// The minimum number of minutes before attempting a server config sync again.
+    static let minimumConfigSyncInterval: TimeInterval = 60 * 60 // 60 minutes
+
```

Comment:
⛏️ I don't see this being used.

Reply:
Ah, looks like I deleted the code that referenced it. Good catch!